README.md

```diff
@@ -85,12 +85,12 @@ For Maven add to your `pom.xml`:
   <dependency>
     <groupId>io.opencensus</groupId>
     <artifactId>opencensus-api</artifactId>
-    <version>0.12.0</version>
+    <version>0.12.2</version>
   </dependency>
   <dependency>
     <groupId>io.opencensus</groupId>
     <artifactId>opencensus-impl</artifactId>
-    <version>0.12.0</version>
+    <version>0.12.2</version>
     <scope>runtime</scope>
   </dependency>
 </dependencies>
```

Comment:
It looks like the script didn't update the Gradle dependencies in the readmes.

Reply:
Good catch. I missed the second command to update gradle versions. Updated.